tools/c7n_azure/c7n_azure/resources/arm.py

```diff
@@ -38,6 +38,7 @@ class ArmTypeInfo(TypeInfo):
     diagnostic_settings_enabled = True
     default_report_fields = (
         'name',
+        'type',
```

Comment:
This actually adds 'type' for all resources.. `azure.armresource` is defined in generic_arm_resource.py

Reply:
Ahh I misunderstood the issue for this one. Yes that makes a lot of sense that we only want to see the type when we are going through generics